functions-and-scope.js

```diff
@@ -41,6 +60,13 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 6.642857142857143
+let sum = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    sum += grades[i];
+    averageOfGrades = sum / grades.length;
```

Comment:
Dit kun je beter buiten de for-loop doen, omdat hij dat nu voor elk cijfer gaat doen